CHANGELOG.rst

```diff
@@ -7,29 +7,35 @@ Change log
 BREAKING CHANGES
 ~~~~~~~~~~~~~~~~
 
-* **migrate to Python3.7**
-* **disable entry points and switch to ``python -m`` calls**
-* **rename ``_universum`` to ``universum``** *(configs need to be fixed)*
+* **migrate to Python3.7** due to Python2 is no longer supported
+    Python3.7 is now :doc:`required <prerequisites>` for Universum to work
+* **disable entry points** and switch to ``python3.7 -m universum`` calls
+    to avoid possible collisions, especially when using :doc:`analyzers <code_report>`
+* **rename `_universum` to `universum`**
+    :class:`~universum.configuration_support.Variations` imports in configs need to be fixed
 
 New features
 ~~~~~~~~~~~~
 
-* **github-handler:** add :doc:`a new Universum mode <args_github_handler>` to serve as a GitHub App
-* **pylint:** allow selecting exact python version
+* **github-handler:** add :doc:`a new Universum mode <github_handler>`
+    to serve as a `GitHub App <https://docs.github.com/en/developers/apps>`__. GitHub Applications,
+    for example, are needed to run checks on newly commmited changes, or on changes to be merged yet
```

Comment:
I think you don't need to add "for example" here.
As far as I understand, GitHub App is needed to create check suite (auto) and check runs and report their status.
We were able to run checks before we implemented GitHub App.

Reply:
For readers I would add that github-handler and other changes in this version allow to implement full workflow for github.